### ISSUE TITLE
Expand make_bmi, cache and ajax with channel, electrode IDs

### ISSUE THREAD

**Issue by m-nolan** [aolabNeuro/brain-python-interface#110]
https://github.com/aolabNeuro/brain-python-interface/blob/11590717e1a1a2d5cc89a0495f02170b1b5f3c08/db/tracker/ajax.py#L49

- Add the electrode IDs and channel IDs to the POST requests in kwargs
- expand cache_and_whatever to accept these inputs
- expand make_bmi to accept these inputs
- save electrode labels in bmi decoder object as attribute, include in saved object structure (not sure how this works for BMI3D)
